components.d.ts

```diff
@@ -1,12 +1,11 @@
 // generated by unplugin-vue-components
 // We suggest you to commit this file into source control
-// Read more: https://github.com/vuejs/vue-next/pull/3399
+// Read more: https://github.com/vuejs/core/pull/3399
 import '@vue/runtime-core'
 
 declare module '@vue/runtime-core' {
   export interface GlobalComponents {
     ISvgClock: typeof import('~icons/svg/clock')['default']
-    ISvgClose: typeof import('~icons/svg/close')['default']
```

Comment:
💭 Since this file is auto-generated each time the dev server does a build, I've been trying to ignore any removals from it, unless we're actually not using an icon anymore. How do you feel about that strategy, or should we just ignore this file altogether?